exercises/strings/countLetter/countLetter.js

```diff
@@ -21,15 +21,26 @@
  * @returns {string} The number of times target letter appears in input string
  */
 
-function countLetter(string, letter) {
+let count = 0;
+function countLetter(string, letter){
   // This is your job. :)
+  for (let targetLetters of string) {
+    if (targetLetters === letter){
+      count = count + 1;
+    }
+  }
+  return count;
 }
 
 if (require.main === module) {
   console.log('Running sanity checks for countLetter:');
 
   // Add your own sanity checks here.
   // How else will you be sure your code does what you think it does?
+  console.log(countLetter('hello', 'l')); // => 2
+  countLetter('Mississippi', 's'); // => 4
```

Comment:
You're not printing out the return value of `countLetter` here, although you're calling the function.

Change to the following:

```js
console.log(countLetter('Mississippi', 's'));
console.log(countLetter('Mississippi', 'x'));
console.log(countLetter('Mississippi', 'm'));
```

You should see some output that surprises you. Can you figure out what's going on?